.github/workflows/ci.yml

```diff
@@ -148,11 +148,70 @@ jobs:
           npm install
           npm run build
           npm run lint
-
+  bump-version:
+    needs: [ template-e2e, template-image-check ]
+    if: github.event_name != 'pull_request' && (startsWith(github.ref, 'refs/tags/v'))
+    name: Bump SDK Version
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18
+      - name: Install tools
+        run: npm i -g semver
+      - name: Configure Git
+        run: |
+          git config --global user.name "$(git log -1 --pretty=format:'%an')"
+          git config --global user.email "$(git log -1 --pretty=format:'%ae')"
+      - name: Set version from tag
+        run: |
+          TAG=$(git describe --tags --abbrev=0)
+          VERSION=${TAG:1}
+          echo "TAG: $TAG"
+          echo "VERSION: $VERSION"
+          echo "NEW_SDK_VERSION=$VERSION" >> $GITHUB_ENV
+      - name: Create a new branch for version bump
+        run: |
+          git checkout -b version-bump-${{ env.NEW_SDK_VERSION }}
+      - name: Bump version and commit
+        run: |
+          ./unifyVersions.js ${{ env.NEW_SDK_VERSION }}
+          npm ci
+          npm i
+          git add .
+          git commit -m "Bump version to ${{ env.NEW_SDK_VERSION }}"
+      - name: Push Changes
+        uses: ad-m/github-push-action@master
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          branch: version-bump-${{ env.NEW_SDK_VERSION }}
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          commit-message: "Bump version to ${{ env.NEW_SDK_VERSION }}"
+          branch: version-bump-"${{ env.NEW_SDK_VERSION }}"
+          delete-branch: true
+          body: "Version bump to ${{ env.NEW_SDK_VERSION }}"
+          labels: version-bump
+          token: ${{ secrets.GITHUB_TOKEN }}
+  auto-merge:
+    needs: [ bump-version ]
+    name: Auto merge
+    runs-on: ubuntu-22.04
+    if: github.event.pull_request.user.login == 'gala-user[bot]'
```

Comment:
maybe we don't need this check here. We could assume that if there's a new tag, everything is fine so the bump PR can be merged automatically without even checking who did that